airdnb/src/main/java/team10/airdnb/member/service/MemberService.java

```diff
@@ -22,14 +25,32 @@ public class MemberService {
 
     private final MemberRepository memberRepository;
 
+    public Member createMember(MemberCreationRequest request) {
+        Member member = request.toEntity();
+
+        validateDuplicateMember(member);
+
+        return memberRepository.save(member);
+    }
+
     public Member registerMember(Member member) {
         validateDuplicateMember(member);
         return memberRepository.save(member);
     }
 
+    public Member loginMember(MemberLoginRequest request) {
+        Member member = findMemberByMemberId(request.email());
+
+        if (!member.getPassword().equals(request.password())) {
```

Comment:
```suggestion
        if (!member.isCorrectPassword(request.password())) {
```

이건 도메인의 메소드로 빠져도 좋을거 같아요

Reply:
안그래도, 항상 로그인 검증 부분에 대해서 고민한 부분이 많았습니다.
처음에 헷갈렸던 부분은, 엔티티 클래스에서 업데이트나 검증과 같은 로직들을 구현해도 되나? 에 대한 의문이 많았는데, 도메인 메소드로 분리하는 개념이더라구요.
감사합니다!